lib/pushmi_pullyu/cli.rb

```diff
@@ -32,39 +32,37 @@ def run
 
   def start_server
     setup_signal_traps
+
     setup_log
     print_banner
 
-    begin
-      run_tick_loop
-    rescue Interrupt
```

Comment:
Feel like rescuing this exception is bad so handle this smarter....

Also allowing for a graceful exit instead of forcing it